src/main/java/org/folio/circulation/domain/FeeFine.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFine {
+  private final JsonObject representation;
+
+  public FeeFine() {
+    representation = null;
+  }
+
+  public FeeFine(JsonObject representation) {
+    this.representation = representation;
+  }
```

Comment:
mod-circulation is trying to move away from keeping the JSON in the domain. Please can this be replaced with fields for the state instead.

Reply:
Done.